hastings-center-report.csl

```diff
@@ -0,0 +1,1259 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" et-al-min="4" et-al-use-first="1" initialize-with=". " page-range-format="chicago" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
```

Comment:
Since you already have `et-al-...` attributes on `<citation/>` and `<bibliography/>`, the `et-al-min="4" et-al-use-first="1"` attributes here don't do anything. Please remove.

Also, if this style is indeed only used in US English, please fix the style to this locale with `<style ... default-locale="en-US"/>` (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#3---style-locale).